compiler/x/amd64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -1668,6 +1668,30 @@ OMR::X86::AMD64::TreeEvaluator::vfirstNonZeroEvaluator(TR::Node *node, TR::CodeG
    return TR::TreeEvaluator::unImpOpEvaluator(node, cg);
    }
 
+TR::Register*
+OMR::X86::AMD64::TreeEvaluator::vmaddEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   return TR::TreeEvaluator::unImpOpEvaluator(node, cg);
```

Comment:
Can you move these out of the AMD64 directory and into the common one. Same issue with the ia32 bit evaluators. The implementation will be the same regardless, so there is no point in duplicating this.

Reply:
done